Ooui.Forms/Renderers/EditorRenderer.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Ooui.Forms.Extensions;
+using System.ComponentModel;
+using System.Diagnostics;
+using Xamarin.Forms;
+
+namespace Ooui.Forms.Renderers
+{
+    public class EditorRenderer : ViewRenderer<Editor, Ooui.TextArea>
+    {
+        private bool _disposed;
+        private Ooui.Color _defaultTextColor;
+
+        static Size initialSize = Size.Zero;
+
+        protected IElementController ElementController => Element as IElementController;
+
+        public override SizeRequest GetDesiredSize(double widthConstraint, double heightConstraint)
```

Comment:
Please use the .editorconfig file so that your formatting matches the project's.

Reply:
Ok!